features/taxonomy.feature

```diff
@@ -0,0 +1,17 @@
+Feature: Manage WordPress taxonomies
+
+  Background:
+    Given a WP install
+
+  @require-wp-3.7
```

Comment:
I assumed it was not necessary to have two different tests just because description field is not present in WP <= 3.7